evap/staff/forms.py

```diff
@@ -120,15 +117,9 @@ def __init__(self, *args, **kwargs):
         super(CourseEmailForm, self).__init__(*args, **kwargs)
 
     def clean(self):
-        self.recipient_groups = []
+        self.recipient_groups = self.cleaned_data.get('recipients')
 
-        if self.cleaned_data.get('sendToAllParticipants'): self.recipient_groups += ['all_participants']
-        if self.cleaned_data.get('sendToDueParticipants'): self.recipient_groups += ['due_participants']
-        if self.cleaned_data.get('sendToResponsible'): self.recipient_groups += ['responsible']
-        if self.cleaned_data.get('sendToEditors'): self.recipient_groups += ['editors']
-        if self.cleaned_data.get('sendToContributors'): self.recipient_groups += ['contributors']
-
-        if len(self.recipient_groups) == 0:
+        if not self.recipient_groups:
```

Comment:
like.

do you know whether we have the option to put custom text into the buttons? right now the text comes from EMAIL_RECIPIENTS, right?